.changeset/search-cars-hide.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search-common': patch
```

Comment:
Kinda unclear to me whether this should be a `patch` or `minor`, given it's a type.  ...But I'm kinda leaning toward minor?  🤷‍♂️

Reply:
Updated to minor 🙂